obspy/clients/filesystem/sds.py

```diff
@@ -194,6 +194,20 @@ def get_waveforms(self, network, station, location, channel, starttime,
             st.merge(merge)
         return st
 
+    def get_waveforms_bulk(self, bulk):
+        """
+        Reads bulk data from a local SeisComP Data Structure (SDS) directory tree.
+
+        Returns a stream object with with the data requested from the local directory.
+
+        :type bulk: list of tuples
+        :param bulk: Information about the requested data.
+        """
+        st = Stream()
+        for bulk_string in bulk:
+            st += self.get_waveforms(*bulk_string)
```

Comment:
So for FDSNWS client, `bulk` can have some variations to its looks. See https://github.com/obspy/obspy/blob/09792d405769ca018f3ec089e9b6f52ffeba126e/obspy/clients/fdsn/client.py#L888-L947

We could think about accepting that string bulk format or file as input here too.. dunno, not sure, so I'd wanna hear out some other opinions first, it's just a thought, and it might make problems later on if some definitive string format for bulk requests comes up. So maybe leave it as is for now.

Reply:
I would be inclined to leave it as is for now as well.